sklearn/linear_model/tests/test_logistic.py

```diff
@@ -2065,6 +2076,47 @@ def test_liblinear_not_stuck():
         clf.fit(X_prep, y)
 
 
+def test_lr_cv_scores_differ_when_sample_weight_is_requested():
+    rng = np.random.RandomState(10)
+    X, y = make_classification(n_samples=10, random_state=rng)
+    sample_weight = np.ones(len(y))
+    sample_weight[: len(y) // 2] = 2
+    kwargs = {"sample_weight": sample_weight}
+
+    with config_context(enable_metadata_routing=True):
+        scorer1 = get_scorer("accuracy")
+        lr_cv1 = LogisticRegressionCV(scoring=scorer1)
+        lr_cv1.fit(X, y, **kwargs)
+
+        scorer2 = get_scorer("accuracy")
+        scorer2.set_score_request(sample_weight=True)
+        lr_cv2 = LogisticRegressionCV(scoring=scorer2)
+        lr_cv2.fit(X, y, **kwargs)
```

Comment:
It would make sense to use the same fixture as here:
https://github.com/scikit-learn/scikit-learn/pull/26789/files#diff-3cdc9913d34276436e69cd506f11cdc77ef286069bd5465aaeeb73ac2bdee318R39-R43

Reply:
I think this is not merged into main right now.

Reply:
You can add it to your PR as well. We will manage the conflict at the time of merge.